src/main/java/com/facebook/openwifirrm/ucentral/models/State.java

```diff
@@ -112,8 +112,21 @@ public class Memory {
 
 	public Unit unit;
 
+	public class Radio {
+		public long active_ms;
+		public long busy_ms;
+		public int channel;
+		public String channel_width;
```

Comment:
note: looks like String is correct
- https://github.com/blogic/ucentral-schema/blob/10373083fb408ce50056b770decefd354c0f820b/renderer/wifi/iface.uc#L20-L28
- https://github.com/blogic/ucentral-schema/blob/85104eb47e25abb103b4f27e037a2392e617593c/system/state.uc#L152